src/test/ui/rust-2018/try-ident.fixed

```diff
@@ -0,0 +1,21 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
```

Comment:
You should just be able to remove this copyright header now (and with the other tests).

Reply:
Ah ok, good to know for the future!

(this'll be a hard habit to shake for me...)